networkx/drawing/nx_pylab.py

```diff
@@ -29,6 +30,10 @@
     spring_layout,
 )
 
+
```

Comment:
This import of warnings is supposed to come on line 19 and delete the extra blank lines at 33-36 leaving a single blank line. (More linting. this time of the imports.)